onward/app/views/series404Test.scala.html

```diff
@@ -0,0 +1,18 @@
+@(surveyUrl: String, metaData: model.MetaData)(implicit request: RequestHeader)
+
+@main(model.SimplePage(metaData)) { } {
+    <div class="l-side-margins">
+        <div class="content">
+            <div class="gs-container">
+                <div class="content__main-column">
+                    <h1 itemprop="headline" class="content__headline">You’ve found our ‘Remind me’ prototype</h1>
+                    <div class="js-article__container u-cf from-content-api" itemprop="articleBody">
+                        <p>Thanks for your interest in this feature.</p>
+                        <p>‘Remind me’ is an early stage prototype at the moment — we want to learn whether Guardian readers will find this useful.</p>
```

Comment:
I asked this a while back - but do we store that you've shown interest to get back to you later?